app/psc/rev-b.toml

```diff
@@ -0,0 +1,356 @@
+name = "psc-b"
+target = "thumbv7em-none-eabihf"
+board = "psc-b"
+chip = "../../chips/stm32h7"
+stacksize = 896
+
+[kernel]
+name = "psc"
+requires = {flash = 32768, ram = 4096}
+#
+# For the kernel (and for any task that logs), we are required to enable
+# either "itm" (denoting logging/panicking via ARM's Instrumentation Trace
+# Macrocell) or "semihosting" (denoting logging/panicking via ARM
+# semihosting).  We are biased to ITM because semihosting is excruciatingly
+# slow (it is breakpoint based) and has an undesirable failure mode if logging
+# output is generated and debugger is not attached (namely, the target stops).
+# If one does choose to change this to semihosting for purposes of
+# development, be sure to also change it in every task of interest.
+#
+features = ["itm"]
+
+[tasks.jefe]
+name = "task-jefe"
+priority = 0
+max-sizes = {flash = 8192, ram = 2048}
+start = true
+features = ["itm"]
+stacksize = 1536
+
+[tasks.jefe.config.allowed-callers]
+set_reset_reason = ["sys"]
+request_reset = ["hiffy"]
+
+[tasks.sys]
+name = "drv-stm32xx-sys"
+features = ["h753"]
+priority = 1
+max-sizes = {flash = 2048, ram = 1024}
+uses = ["rcc", "gpios1", "gpios2", "gpios3"]
+start = true
+task-slots = ["jefe"]
+
+[tasks.spi4_driver]
+name = "drv-stm32h7-spi-server"
+priority = 2
+max-sizes = {flash = 16384, ram = 2048}
+features = ["spi4", "h753"]
+uses = ["spi4"]
+start = true
+interrupts = {"spi4.irq" = 1}
+stacksize = 872
+task-slots = ["sys"]
+
+[tasks.spi4_driver.config.spi]
+global_config = "spi4"
+
+[tasks.spi2_driver]
+name = "drv-stm32h7-spi-server"
+priority = 2
+max-sizes = {flash = 16384, ram = 2048}
+features = ["spi2", "h753"]
+uses = ["spi2"]
+start = true
+interrupts = {"spi2.irq" = 1}
+stacksize = 872
+task-slots = ["sys"]
+
+[tasks.spi2_driver.config.spi]
+global_config = "spi2"
+
+[tasks.i2c_driver]
+name = "drv-stm32xx-i2c-server"
+features = ["h753", "itm"]
+priority = 2
+max-sizes = {flash = 16384, ram = 2048}
+uses = ["i2c2", "i2c3", "i2c4"]
```

Comment:
Is this right? I only see two i2c ports on the SP being used:

* I2C_TO_SP_LOCAL
* I2C_TP_TO_PS

It doesn't appear I2C4 is actually used.

Reply:
Correct, there are only two I2C ports on the SP being used. The third port was previously used to talk to some I2C GPIO expanders, which we deleted.

Reply:
👍🏻  I deleted it lower down, but forgot to remove it from the driver task; fixed!